db/migrate/20200527142734_remove_kithe_derivatives_table.rb

```diff
@@ -1,5 +1,6 @@
 class RemoveKitheDerivativesTable < ActiveRecord::Migration[6.0]
   def change
-    drop_table :kithe_derivatives
+    drop_table :kithe_derivatives do |t|
+    end
```

Comment:
how'd this end up getting changed in this PR? The change is basically no-op I think, I don't think it'll hurt, also isn't necessary, just not sure how it crept in here

Reply:
I added it in because your migration, which was chronologically *after* my migration, had been preventing me from reversing my migration. No longer needed.